lib/cb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.1'
+  VERSION = '16.2.2'
```

Comment:
:+1: yeah this is totally backward compatible, I've officially talked myself into it. Good call ultraman!